source/connect/ssh-connection.txt

```diff
@@ -11,3 +11,157 @@ Proxy / SSH Tunnel Connection Tab
    :backlinks: none
    :depth: 1
    :class: singlecol
+
+The :guilabel:`Proxy / SSH Tunnel` tab allows you to connect your deployment using 
```

Comment:
Should this be "...to connect to your..."? Ditto throughout this update for similar wording.

Reply:
Done.